src/tourofclojure/pages/macros.clj

```diff
@@ -6,18 +6,42 @@
 (def code
   (slurp (io/resource "public/pages/code/macros.clj")))
 
+(def code-result
+  (slurp (io/resource "public/pages/code/macros-result.clj")))
+
 (defn desc
   [previous next lang]
   (condp = lang
     "fr" [:div
           [:h2 "Macros"]
-          [:p "TODO"]
-          ]
+          [:p "Clojure dispose d'un système de macros qui permet au développeur d'étendre les fonctionnalités du langage.
+           Les macros permettent d'introduire des nouveaux éléments sans dépendre d'un changement dans le coeur."]
```

Comment:
Je reformulerai en quelque chose comme "Clojure dispo d'un système de macro permettant de créer des abstractions complexes pouvant servir notamment à ajouter des fonctionnalités au langage. 
Par exemple, core.async (lien vers la partie core.async) est majoritairement écrit avec des macros.

Reply:
Je vais modifier dans ce sens là.
Toutefois, même si core.async est un exemple parfait, beaucoup d'éléments présents dans le langage sont des macros également (cond, condp, for, when..).

Du coup, contre-proposition:
"En réalité, beaucoup d'éléments présentés dans ce tutoriel sont en réalité des macros et non pas des fonctions (cond, condp, for, when..) et certaines librairies comme core.async est majoritement écrite avec des macros"

Ca pourrait t'aller?

Reply:
ça me va ;)